draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -740,6 +741,14 @@ The proxy can do RADIUS/UDP to some servers and RADIUS/(D)TLS to others.
 Delegation of responsibilities and separation of tasks are important security principles.
 By moving all RADIUS/(D)TLS knowledge to a (D)TLS-aware proxy, security analysis becomes simpler, and enforcement of correct security becomes easier.
 
+## Network Address Translation
+
+Network Address Translation (NAT) is fundamentally incompatible with RADIUS/UDP. RADIUS/UDP uses the source IP address to determine the shared secret for the client, and NAT hides many clients behind one source IP address. As a result, RADIUS/UDP clients cannot be located behind a NAT gateway.
+
```

Comment:
This statement is fundametally wrong. RADIUS/UDP works perfeclty fine behind a NAT, as many Wifi deployments where each individual AP talks to the radius server directly demonstrates. The only caveat is that all RADIUS clients behind a nat MUST use the same shared secret.

Reply:
The "fundamentally incompatible" phrase is not about whether it works, but whether it breaks the security model of RADIUS.  Having multiple clients behind a NAT might "work", but it has not been standardized in the RFCs.

Having multiple clients share the same secret is bad security practice, and should not be recommended by a standard,.